osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/InvocationCounterManager.java

```diff
@@ -66,11 +66,10 @@ private void initializeWithInvocationCounterStoredOnDevice(final DlmsDevice devi
         }
     }
 
-    @SuppressWarnings("squid:S1905") // Casting to Number is necessary here.
     private long getInvocationCounter(final DlmsConnectionManager connectionManager) throws FunctionalException {
         return ((Number) this.dlmsHelper
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "Number". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1905)